examples/AllTests/EventDispatcherTest.cpp

```diff
@@ -67,12 +67,12 @@ TEST_GROUP(EventDispatcher)
     ObserverMock observer2;
     EventComparator eventComparator;
 
-    void setup()
+    TEST_SETUP()
```

Comment:
Could we keep the "void setup ()" instead? The "TEST_SETUP" macro is there mostly for backwards compatability as the "void setup" makes it clearer what is happening.

Reply:
sorry. lost track of this one. done.